src/api/scanned-codes.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+parent: API Reference
+title: Scanned Codes
+permalink: /api/scanned-codes
+---
+
+# Scanned Codes
+
+A scanned code is a barcode that a merchant scans. The code can be used to create a [Payment Request][].
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Scanned Code
+
+{% h4 Mandatory Fields %}
```

Comment:
This has changed. Can we follow standards we set out in the [Patron Code](https://docs.centrapay.com/api/patron-codes)?

If we call the table "Fields" then this would fit that description.

```suggestion
{% h4 Fields %}
```

Reply:
- [ ] Tick this
- [ ] Never tick this